recipes/uvicorn-tschaume/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "uvicorn-tschaume" %}
+{% set version = "0.19.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/uvicorn_tschaume-{{ version }}.tar.gz
+  sha256: bed9de09b24c87abec4374644f73fc19859e3ebb137f5277632dda3b6c16ba45
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
```

I think this can be made noarch if you do:
```suggestion
  number: 0
  noarch: python
    entry_points:
    - bsdiff4 = uvicorn.main:main
```